lib/generators/rolemodel/lograge/lograge_generator.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require_relative '../../bundler_helpers'
+
+module Rolemodel
+  class LogrageGenerator < Rails::Generators::Base
+    include Rolemodel::BundlerHelpers
+    source_root File.expand_path('templates', __dir__)
+
+    def install_lograge
+      run 'bundle add lograge'
```

Comment:
If we're including the `Rolemodel::BundlerHelpers`, does the syntax of this line change?